host-configs/llnl/blueos_3_ppc64le_ib_p9/clang@upstream_link_with_nvcc.cmake

```diff
@@ -52,24 +51,21 @@ set(ENABLE_OPENMP ON CACHE BOOL "")
 set(OMP_HOME ${CLANG_HOME}/ibm/omprtl)
 set(BLT_OPENMP_LINK_FLAGS "-Xlinker -rpath -Xlinker ${OMP_HOME}/lib -L${OMP_HOME}/lib -lomp -lomptarget-nvptx" CACHE STRING "")
 
-
 #------------------------------------------------------------------------------
 # CUDA support
 #------------------------------------------------------------------------------
 set(ENABLE_CUDA ON CACHE BOOL "")
 
 set(CUDA_TOOLKIT_ROOT_DIR "/usr/tce/packages/cuda/cuda-9.2.148" CACHE PATH "")
 set(CMAKE_CUDA_COMPILER "${CUDA_TOOLKIT_ROOT_DIR}/bin/nvcc" CACHE PATH "")
-set(CMAKE_CUDA_HOST_COMPILER ${MPI_CXX_COMPILER} CACHE PATH "")
+set(CMAKE_CUDA_HOST_COMPILER ${CMAKE_CXX_COMPILER} CACHE PATH "")
 
 set (_cuda_arch "sm_60")
 set (CMAKE_CUDA_FLAGS "-restrict -arch ${_cuda_arch} -std=c++11 --expt-extended-lambda -G" CACHE STRING "" )
 
 set (CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "" )
 set (CUDA_LINK_WITH_NVCC ON CACHE BOOL "")
-# set the link flags manually since nvcc will link (and not have the wrappers knowledge)
-# on ray - can figure out your equivalant flags by doing mpicc -vvvv
-set (CMAKE_CUDA_LINK_FLAGS "-Xlinker -rpath -Xlinker ${MPI_HOME}/lib -Xlinker -rpath -Xlinker ${CLANG_HOME}/ibm/lib:/usr/tce/packages/gcc/gcc-4.9.3/lib64 -L${MPI_HOME}/lib/ -lmpi_ibm" CACHE STRING "")
+
```

Comment:
nice!

Reply:
I'm assuming you tested this against all the relevant host-configs.

Reply:
I ran both blueos host-configs on rzansel and rzmanta.  They passed on cmake 3.9.2, but failed on 3.12.1 and 3.14.5.  I verified we were unable to pass tests on BLT's develop on those versions.